### DISCUSSION TITLE
Combine debWV and MRFSS length data for comps?

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#56] [Q&A]
Is there any issue with combining the deb wilson vandenberg length data with mrfss era length data other than the following?
- possible risk of duplicates from sampling on vessels through Deb, and then again dockside through MRFSS 
- Known issue in 1997-1998 of deb data being in mrfss ([see #5](https://github.com/pfmc-assessments/california-data/discussions/5) in california-data repo)

@melissamonk-NOAA and @chantelwetzel-noaa, I know you are planning on replacing PC mode lengths in MRFSS with the data from deb for 1987-1998 for the northern model. I was planning on doing something similar, but was asked why not combine, and I dont know. For quillback and copper we added these to the rest of the recreational lengths, while for lingcod we had a separate deb index fleet. 

Unless the risk of having MRFSS sample deb boats is large, it would seem like combining MRFSS and deb is sensible.

**Comment by chantelwetzel-noaa** [comment]
I don't think we had the Deb Wilson Vandenberg lengths for the data-moderate assessments. I definitely did not have them for copper rockfish. As I mentioned in our previous discussion, we found clear overlap in 1997-98 where the data in MRFSS were the exact the same records as those in Deb's data.  There has been long standing concerns about having some level of duplication across these data sources. Hence we are opting to only use Deb's for these years since the sample size is significantly higher.

**Comment by brianlangseth-NOAA** [comment]
Other than years with known issues (1997-1998), and possibility that there could be overlap in other years if dockside samples Deb fish that are retained, there appears no issue with using both. Talked with Melissa and she suggested reading in each as its own comp so that can easily test effect of one or the other, or both.
